velox/exec/Task.h

```diff
@@ -254,6 +254,19 @@ class Task : public std::enable_shared_from_this<Task> {
   /// structure.
   TaskStats taskStats() const;
 
+  /// Information about an operator call that helps debugging stuck calls.
+  struct OpCallInfo {
+    size_t durationMs;
+    int32_t tid;
+    int32_t opId;
+    std::string taskId;
+  };
+
+  /// Collect long running operator calls across all drivers in this task.
```

Comment:
/// Collects long running operator calls across all drivers in this task which execution time is longer than 'thresholdDurationMs'.